docker/db-migration/migrations/013-remove-flow-group-id-field.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
```

Comment:
Why we have this file?

Reply:
Rebase artifact. This file has been deleted.